index.html

```diff
@@ -22,26 +24,189 @@
     <main>
       <section class="header-banner">
         <h2 class="name">I'm Oscar<br>Glad to see<br>you!</h2>
-        <p class="introduction">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <p class="introduction">I'm a software developer! I can help you build a product,
+           feature or website Look through some of my work and experience! 
+           If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
 
         <a href="https://linkedin.com/in/OscarFM-dev" class="connect-text">LET'S CONNECT</a>
         <ul class="social-media-icons">
           <li>
-            <a href="https://linkedin.com/in/OscarFM-dev"><img src="./images/linkedin.png" alt="linkedin logo" class="linkedin-logo"></a>
+            <a href="https://linkedin.com/in/OscarFM-dev">
+              <img src="./images/header-section/linkedin.png" alt="linkedin logo" class="linkedin-logo">
+            </a>
           </li>
           <li>
-            <a href="https://github.com/OscarFM-dev"><img src="./images/github.png" alt="github logo"></a>
+            <a href="https://github.com/OscarFM-dev">
+              <img src="./images/header-section/github.png" alt="github logo">
+            </a>
           </li>
           <li>
-            <a href="https://twitter.com/OscarFMdev"><img src="./images/twitter.png" alt="twitter logo"></a>
+            <a href="https://twitter.com/OscarFMdev">
+              <img src="./images/header-section/twitter.png" alt="twitter logo">
+            </a>
           </li>
           <li>
-            <a href="https://www.instagram.com/oscarfmdev/"><img src="./images/peace.png" alt="peace logo"></a>
+            <a href="https://www.instagram.com/oscarfmdev/">
+              <img src="./images/header-section/peace.png" alt="peace logo">
+            </a>
           </li>
         </ul>
       </section>
+
+      <section class="works">
+
+        <div class="project">
+          <img src="./images/works-section/project1.png" alt="First project image">
```

Comment:
- [ ] Kindly, order the images of your design to look similar to the Figma template. :octocat: 

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Reply:
Sorry, I didn't understand this part, could you explain me, please? do you mean the image sizes must be in pixels instead of responsive?